### ISSUE TITLE
Missing <interval> block in user.j2

### ISSUE THREAD

**Issue by DeanThompson** [AlexeySetevoi/ansible-clickhouse#34]
Below is the current version:

```
    {% for quota in clickhouse_quotas_default %}
    <{{ quota.name }}>
    {% for interval in quota.intervals %}
        <duration>{{ interval.duration }}</duration>
        <queries>{{ interval.queries }}</queries>
        <errors>{{ interval.errors }}</errors>
        <result_rows>{{ interval.result_rows }}</result_rows>
        <read_rows>{{ interval.read_rows }}</read_rows>
        <execution_time>{{ interval.execution_time }}</execution_time>
    {% endfor %}
    </{{ quota.name }}>
    {% endfor %}
```

It should be:

```
    {% for quota in clickhouse_quotas_default %}
    <{{ quota.name }}>
    {% for interval in quota.intervals %}
        <interval>
            <duration>{{ interval.duration }}</duration>
            <queries>{{ interval.queries }}</queries>
            <errors>{{ interval.errors }}</errors>
            <result_rows>{{ interval.result_rows }}</result_rows>
            <read_rows>{{ interval.read_rows }}</read_rows>
            <execution_time>{{ interval.execution_time }}</execution_time>
        </interval>
    {% endfor %}
```

The `clickhouse_quotas_custom` section should also be modified.

**Comment by TDenisM** [comment]
The problem is still here.
Also I sugget to make some changes in users.j2 file in Quota section:
```
<!-- Quotas. -->
    <quotas>
        <!-- Default quotas. -->
    {% for quota in clickhouse_quotas_default %}
    <{{ quota.name }}>
    {% for interval in quota.intervals %}
    <interval>
    {% for key, value in interval.items() %}
        <{{ key }}>{{ value }}</{{ key }}>
    {% endfor %}
    </interval>
    {% endfor %}
    </{{ quota.name }}>
    {% endfor %}
    <!-- Custom quotas -->
    {% for quota in clickhouse_quotas_custom %}
    <{{ quota.name }}>
    {% for interval in quota.intervals %}
    <interval>
    {% for key, value in interval.items() %}
        <{{ key }}>{{ value }}</{{ key }}>
    {% endfor %}
    </interval>
    {% endfor %}
    </{{ quota.name }}>
    {% endfor %}
    </quotas>
```
It makes it  more flexible.
You need to add ```clickhouse_quotas_custom``` to defaults:
```
clickhouse_quotas_custom:
 - { name: "user", intervals: "{{ clickhouse_quotas_intervals_custom }}", comment: "My custom quotas" }
```

**Comment by AntonFriberg** [comment]
The original issue seems solved by #65 ?
